tests/validation/tests/v3_api/test_import_cluster.py

```diff
@@ -6,34 +6,44 @@
 DATA_SUBDIR = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                            'resource')
 AWS_SSH_KEY_NAME = os.environ.get("AWS_SSH_KEY_NAME")
+AWS_NODE_COUNT = int(os.environ.get("AWS_NODE_COUNT", 3))
 
 
 def test_import_rke_cluster():
 
     client = get_user_client()
 
-    # Create nodes in AWS
-    aws_nodes = create_nodes()
+    # Create AWS nodes for the cluster
+    aws_nodes = \
+        AmazonWebServices().create_multiple_nodes(
+            AWS_NODE_COUNT, random_test_name("rketestcustom"),
+            wait_for_ready=True)
+    assert len(aws_nodes) == AWS_NODE_COUNT
```

Comment:
We should not rely on AWS_NODE_COUNT since we can only support provisioning up to 3 nodes .

Reply:
We can rely on AWS_NODE_COUNT and changes will be made to support it.